deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml

```diff
@@ -170,18 +170,6 @@ spec:
                 app: smart-gateway-operator
             spec:
               containers:
-              - command:
```

Comment:
This is because it's no longer supported by the new version of operator-sdk, right?

Reply:
That's exactly it. The ao-logs stuff was already noting that it was deprecated in 0.16.0. In the latest operator-sdk (0.19.4 in our case, so not really _latest_... you get that in `4.7`) if you generate a new instance, then you don't end up with any of this. I used the example to make sure I matched what would be the expected output.